docs/UserGuide.md

```diff
@@ -306,6 +307,11 @@ Alternatively, you can use csv config files to customize the analysis as before
 * `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
 * `author-config.csv`: Add one row for each author in each repo you want to analyze
 
+
+## Publishing report to GitHub Pages
+
+Follow this [guide](PublishingGuide.md) to publish your report to GitHub Pages.
```

Comment:
For discussion:

I am not sure if the title should be `Publishing report to GitHub Pages` because it kinds of make me think why do I even need to read it? Like if I know what a GH Pages is, I would just run the report and push it to my own GH Pages.

Perhaps `Using CI Service` would be more accurate? @damithc what do you think?

Reply:
`Using Travis to Auto-Publish the Report on GitHub Pages`?

Reply:
Or `Auto-Update`